betty/cropper/models.py

```diff
@@ -324,11 +324,15 @@ def clear_crops(self, ratios=None):
                 paths += [self.get_absolute_url(ratio=ratio_slug, width=width, format=format)
                           for format in ["png", "jpg"]
                           for width in settings.BETTY_WIDTHS]
+            if self.animated:
+                for format in ['gif', 'jpg']:
```

Comment:
should these kinds of lists be configurable? or at the very leasts constants set in the module?